students/wachulec_maria/lesson_9_reading_and_writing_files/regex_search.py

```diff
@@ -0,0 +1,26 @@
+import os
+import re
+
+
+def search_in_file(file, reg):
+    txt_file = open(os.path.join(os.getcwd(), file))
+    text = txt_file.read()
+    regex = (re.findall(reg, text))
+    txt_file.close()
```

Comment:
I'm aware that it wasn't mentioned in Automate Boring stuff, but for handling files/io the much more preferred way is to use context managers (`with`)

https://www.python.org/dev/peps/pep-0343/
https://stackoverflow.com/questions/1369526/what-is-the-python-keyword-with-used-for